hack/xref-helpmsgs-manpages

```diff
@@ -619,6 +619,8 @@ sub _is_valid_external_link {
         return 1 if $link eq "https://www.freedesktop.org/software/systemd/man/$base.html";
     }
 
+    return 1 if $link eq "https://passt.top/builds/latest/web/passt.1.html";
```

Comment:
Another commit that needs to be squashed; please fold this into 619e918ef4e05f3c4b50e47a4375d7648de5a553

Reply:
Sure, that belongs to documentation changes, I forgot to squash it. I'll do that.

Reply:
Squashed.